src/mumble/main.cpp

```diff
@@ -125,7 +125,24 @@ int main(int argc, char **argv) {
 	}
 #endif
 
-	Global::g_global_struct = new Global();
+	QStringList args = a.arguments();
+	if (args.contains(QLatin1String("-c")) || args.contains(QLatin1String("--config"))) {
+		int index = std::max(args.indexOf(QLatin1String("-c")), args.indexOf(QLatin1String("-c")));
```

Comment:
What are you trying to do here? To me it appears as if you are doing the same `indexOf` operation twice and then taking the maximum which doesn't make much sense since both operations should return the exact same value...

Reply:
Ooof. One of those is supposed to be `--config` :sweat_smile:.